verificarlo.in.in

```diff
@@ -199,6 +217,7 @@ if __name__ == "__main__":
     parser.add_argument('-static', '--static', action='store_true', help='produce a static binary')
     parser.add_argument('--verbose', action='store_true', help='verbose output')
     parser.add_argument('--inst-fcmp', action='store_true', help='instrument floating point comparisons')
+    parser.add_argument('--inst-func', action='store_true', help='instrument floating point functions')
```

Comment:
You should change a little bit the help section since it can be confusing.

Reply:
I'm agree, this has been replaced by "instrument functions" because now a function is instrumented even if she don't use "float" or "double".